src/app/series/directives/series-podcast.component.html

```diff
@@ -1,8 +1,13 @@
 <form [ngSwitch]="state">
 
   <publish-fancy-field label="Series Podcast" *ngSwitchCase="'new'">
-    <div class="fancy-hint">You must save your series before you can create
-      a podcast distribution for it.</div>
+    <div class="fancy-hint">The series itself must be created before a podcast distribution
+      can be created for it.</div>
```

Comment:
I changed this language because a) I thought it should say create to match the button rather than save and b) it's said that using "You" or "You must" in alerts or error messages can be interpreted as insulting or condescending.